content/recipes/introduction/public-kg-resource-integration.md

```diff
@@ -409,54 +364,43 @@ result = pipeline(
 )
 ```
 
-To see an example of how to use the library, we highly recommended reading the following Medium blog post:
-
-https://towardsdatascience.com/knowledge-graph-completion-with-pykeen-and-neo4j-6bca734edf43
-
-and consult the associated jupyter notebook, which shows how to use your own dataset, understand the evaluation,
-and make novel link predictions:
-
-https://github.com/tomasonjo/blogs/blob/master/pykeen/Hetionet%20-%20RotatE.ipynb
-
-
+To see an example of how to use the library, we highly recommended reading the [Medium blog post](https://towardsdatascience.com/knowledge-graph-completion-with-pykeen-and-neo4j-6bca734edf43) and consult the associated jupyter notebook, which shows how to use your own dataset, understand the evaluation, and make novel link predictions:
 
+Link Prediction example notebook: https://github.com/tomasonjo/blogs/blob/master/pykeen/Hetionet%20-%20RotatE.ipynb
 
 ---
```

Comment:
is PDBe-KG https://www.ebi.ac.uk/pdbe/pdbe-kb/graph-download also in the scope of this article?